routes/api.js

```diff
@@ -144,7 +144,7 @@ function fillExamples() {
 // Set request options
 var options = {
     "headers": {
-        "user-agent": "dcos/1.9",
+        "user-agent": "dcos/1.10",
```

Comment:
Yes. DC/OS 1.10 and now DC/OS 1.11 should show all of the Universe packages. I would also change the accept media type to `application/vnd.dcos.universe.repo+json;charset=utf-8;version=v4`. That is the latest media type for the content of Universe.

Reply:
Thx

Reply:
@jsancio is there any other way to query the universe? Seems strange to use the `user-agent` to query for compatible packages esp. in the universe-browser use-case.

Reply:
It is actually standard for HTTP servers to do some processing (filtering, redirect, etc) based on the user-agent. If you would like to avoid the redirect server completely you can use this URL instead: `s3://downloads.mesosphere.io/universe/repo/repo-up-to-1.11.json` or `https://downloads.mesosphere.come/universe/repo/repo-up-to-1.11.json`.

As part of this CI (https://teamcity.mesosphere.io/viewLog.html?buildId=825644&buildTypeId=ClosedSource_Dcos_Universe_ProductionUniverse_PublishS3&tab=buildLog&_focus=62#_state=77,45) we generate the full universe. It looks like we just don't upload it.

Having said that, the suggestions above are not part of the supported API. In the not to distant future we will make changes to Universe that will break any client that doesn't use `https://universe.mesosphere.com/repo`

Reply:
"In HTTP, the User-Agent string is often used for content negotiation, where the origin server selects suitable content or operating parameters for the response. For example, the User-Agent string might be used by a web server to choose variants based on the known capabilities of a particular version of client software." from https://en.wikipedia.org/wiki/User_agent

Reply:
@jsancio I agree that it's not uncommon to use the `user-agent` header to selects suitable content or perform redirects to workaround reported problems and limitations, as documented in RFC 7231, but the `user-agent` header should contain information about the user agent originating the request.  
In case of DC/OS requesting compatible packages it's perfectly fine to select them based on the header, but for the universe browser I'd expect a parameter to select packages. What do you think?